src/components/PassageDataTable.tsx

```diff
@@ -153,6 +156,18 @@ export const SelectSharedResource = (props: IProps) => {
     [plan]
   );
 
+  useEffect(() => {
+    var val = value ?? -1;
+    if (val >= 0) {
+      if (val < data.length) {
+        if (checks.findIndex((r) => r === val) < 0) setChecks([val]);
+      } else {
+        console.log('not in my list');
```

Comment:
I am not sure I understand this message. It is not an error or reported to the error service. Is it assumed that it will never happen?